src/src/pages/topics/index.js

```diff
@@ -216,8 +236,8 @@ function Topics() {
         ) : (
           <>
             <MaterialReactTable columns={columns} data={filteredData}
-              positionExpandColumn="last"
-              displayColumnDefOptions={{                
+              positionExpandColumn="left"
```

Comment:
Add a comment that this is a hack :) Is there a better way to turn off this feature?

Reply:
Actually adding the initial state of ordering columns fixed that for us. 

```
 initialState={{
                columnOrder: [
                  'name',
                  'arrow',
                ],
              }}
```